andela/theoden-cfh
# #####151438242 OAuth authentication

#### What does this PR do?
Allows users to access the game through social media logins (Facebook, Twitter and github)

#### Description of Task to be completed?
Users should be able to gain access to the game through the social media buttons on both login and sign up pages (Twitter, Facebook, Github)

#### How should this be manually tested?
On sign up user should click on his preferred social media platform, they will be directed to choose an avatar and then to the gaming screen.

#### Any background context you want to provide?
-Update your development.json with the appropriate credentials. 
-On deployment to heroku , production.json will need to be updated with appropriate credentials for users to sign in.

#### What are the relevant pivotal tracker stories?
#151136245, #151136244

[Finishes #151438242]
